index.html

```diff
@@ -1,11 +1,53 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+    <link rel="stylesheet" type="text/css" href="style/index.css">
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
 </head>
 <body>
     
+<header>
+    <div class="sitename">
+        <h1>Jonathan Richardson</h1>
+    </div>
+    <div class="links">
+        <nav>
+        <a href="index.html">Home</a>
```

Comment:
awesome job building out your navigation and linking them to the corresponding pages!